config/example/config.toml

```diff
@@ -263,7 +263,7 @@ gamemode_probabilities = [
 	{ gamemode = "changeling", probability = 0 },
 	{ gamemode = "cult", probability = 3 },
 	{ gamemode = "extend-a-traitormongous", probability = 2 }, # Autotraitor
-	{ gamemode = "extended", probability = 0 },
+	{ gamemode = "extended", probability = 1 },
```

Comment:
is this supposed to stay in the final version?

Reply:
nope, need to undo

Reply:
undid